### PR TITLE
Added one more task based on movie details

### PR DESCRIPTION
**Pull request by yadavtushita**

### CODE REVIEW

**Comment by yadavtushita** [comment]
Pushed movie details code directly to main by mistake, this is the commit: https://github.com/nfeagh/react-flask-app/commit/35eb52785bea9751420f4977c47c839efa3917cf

File: inbox/email_J_11.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Hi Team,
+
+Our users have started sending in requests to understand whether the movie is rated G, PG..... and they also want some more information on how long the movie is. Could you take a look and see if it is possible to display these details?
+
+Jeenal
+
+## Tips
+Take a look at the [mocked movide details](src/mocks/movieDetails.js) to see if further details are available and what information is relevant to fulfill this request.
```
Comment (KushTurner) [inline inbox/email_J_11.md]: movide -> movie
fulfill -> fulfil

Comment (yadavtushita) [reply inbox/email_J_11.md]: missed that, thanks!

File: inbox/email_J_11.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Hi Team,
+
+Our users have started sending in requests to understand whether the movie is rated G, PG..... and they also want some more information on how long the movie is. Could you take a look and see if it is possible to display these details?
```
Comment (KushTurner) [inline inbox/email_J_11.md]: please can we make this "..." instead of "....." ? and also capitalize and.

sorry 😂

Comment (yadavtushita) [reply inbox/email_J_11.md]: of course check now XD

**Pull request merged**